### DISCUSSION TITLE
ownership verification for inscription holders?

### DISCUSSION THREAD

**Discussion by punk3700** [ordinals/ord#1645] [General]
hey.  we just released a long-form gen art collection (called [timechain](https://generative.xyz/generative/1000001)) on ordinals. we have 20+ mints so far.  super excited!

the holders are asking if we can set up a private chat on discord for them.  

how do you verify if someone is holding or not holding a specific inscription?

thanks!

**Comment by raphjaph** [comment]
Haven't implemented yet but have a look at #1462.

**Comment by reinamora137** [comment]
With BIP-322 support in bitcoin core users can sign messages. 

[https://github.com/bitcoin/bitcoin/pull/24058
](https://github.com/bitcoin/bitcoin/pull/24058)

The guys over at ordsea.io are doing this for verification of listing on their site: 

[https://www.ordsea.io/how](https://www.ordsea.io/how)

Curious to hear more feedback on this functionality. Will the support in the ord wallet require the BIP-322 support in the users bitcoin-core?

**Comment by diegoxter** [comment]
You could, potentially, prompt the user for an ordinal ID (think of it as a login), have the user sign a message for authentification and then verify against any database out there. 

I have done this for an ordinal project I'm working
